test/Nanoid.Test/Nanoid.Test.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;netcoreapp3.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0;netcoreapp3.0;net7</TargetFrameworks>
```

Comment:
Added in order to run the unit tests and make sure they still pass when using `Int32.LeadingZeroCount` in .net7 and above.